docs/en/operations/system-tables/user_processes.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: /en/operations/system-tables/user_processes
+---
+# user_processes
+
+This system table is used for implementing the `SHOW USER PROCESSES` query.
```

Comment:
Do we really need this query type? 

Actually system table is accessible regardless of this query, so we have to write here something more generic about this table itself (e.g. "table with information about user processes ...") and just add "see also" link to the query doc entry

Doc entry for `SHOW USER PROCESSES` is also required if we need the query.

Reply:
I removed the query.